dist/flags.py

```diff
@@ -8,6 +8,9 @@
 ###################################################
 # Internal routine flag declarations
 ###################################################
+    'diagnostic_timing_stress' : [
+        'DIAGNOSTIC_CHECKPOINT_SLOW',
```

Comment:
I think the `DIAGNOSTIC` word is the less useful descriptor for this flag. I'd prefer it be called `TIMING_STRESS_CHECKPOINT_SLOW`. Since this should be considered a "timing stress" mode.

This comment propagates to several different places in the code, they should all be changed.